README.md

```diff
@@ -1,29 +1,32 @@
 # React Native Test App
 
-[![Open in Visual Studio Code](https://img.shields.io/static/v1?logo=visualstudiocode&label=&message=Open%20in%20Visual%20Studio%20Code&color=007acc&labelColor=444444&logoColor=007acc)](https://vscode.dev/github/microsoft/react-native-test-app)
+![Contributions Welcome](https://img.shields.io/badge/contributions-welcome-brightgreen) [![Open in Visual Studio Code](https://img.shields.io/static/v1?logo=visualstudiocode&label=&message=Open%20in%20Visual%20Studio%20Code&color=007acc&labelColor=444444&logoColor=007acc)](https://vscode.dev/github/microsoft/react-native-test-app)
 [![build](https://github.com/microsoft/react-native-test-app/actions/workflows/build.yml/badge.svg?event=push)](https://github.com/microsoft/react-native-test-app/actions/workflows/build.yml)
 [![npm version](https://img.shields.io/npm/v/react-native-test-app)](https://www.npmjs.com/package/react-native-test-app)
 
-- [Quick Start](#quick-start)
+- [Quick Start](https://github.com/microsoft/react-native-test-app/wiki/Quick-Start)
 - [Configuring the Test App](#configuring-the-test-app)
+- [Migrate an Existing Test App](https://github.com/microsoft/react-native-test-app/wiki/Migrate-an-Existing-Test-App)
 - [Known Issues](#known-issues)
 
-React Native Test App provides test apps for all platforms as a package. It
+React Native Test App (RNTA) provides test apps for all platforms as a package. It
 handles the native bits for you so you can focus on what's important: your
 product.
 
 If you want to learn how RNTA is used at Microsoft, and see a demo of how to add
 it to an existing library - you can watch the
 ["Improve all the repos – exploring Microsoft’s DevExp"](https://youtu.be/DAEnPV78rQc?t=499)
-talk by @kelset and @tido64 from React Native Europe 2021.
+talk by [@kelset](https://github.com/kelset) and [@tido64](https://github.com/tido64) from React Native Europe 2021.
 
-For more about
+Read more about
 [the motivation](https://github.com/microsoft/react-native-test-app/wiki#motivation)
 and [the design](https://github.com/microsoft/react-native-test-app/wiki/Design)
-of this tool, you can refer to the wiki.
+of this tool.
```

Comment:
Can we rephrase this so that it still makes sense even without the hyperlinks?

Reply:
Same for all the new lines below. I like to have proper sentences for screen readers etc.

Reply:
@tido64 check it out now, lmk if it's better